### PR TITLE
Improving git

### PR DESCRIPTION
**Pull request by Poncheele**
Tutorial to fix de bugg "Le terme «conda» n'est pas reconnu comme nom d'applet de commande..."

### CODE REVIEW

**Review by josephsalmon — COMMENTED** [review]
Thank you for the update.
The name of the file is not a valid names (contains spaces).
Could you udpate it for instance with underscores instead (Debug_path....)?

**Comment by Poncheele** [comment]
Done

File: Windows/Debbug path conda windows.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# How to fix "Le terme «conda» n'est pas reconnu comme nom d'applet de commande...".
+
+### Step 1 : Uninstall anaconda, then install again.
```
Comment (josephsalmon) [inline Windows/Debbug path conda windows.md]: this part should be ## not ###

File: Windows/Debbug path conda windows.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+When this window appear : 
+<!-- <img src="files/images/python-screenshot.jpg" width="600"> -->
+
+<img src="https://raw.github.com/Poncheele/doc/master/Anaconda.png" width="600">
+
+Check the box "Add Anaconda to my Path environment variable"
+
+### step 2 : Create your environment
```
Comment (josephsalmon) [inline Windows/Debbug path conda windows.md]: idem

File: Windows/Debbug path conda windows.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+#### Open Anaconda Prompt
+
+Write the following commands :
+
+    conda create -n hmma238 matplotlib
+
+    conda activate hmma238
+
+### Step 3 : Check if it works
```
Comment (josephsalmon) [inline Windows/Debbug path conda windows.md]: idem

File: Windows/Debbug path conda windows.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+When this window appear : 
+<!-- <img src="files/images/python-screenshot.jpg" width="600"> -->
+
+<img src="https://raw.github.com/Poncheele/doc/master/Anaconda.png" width="600">
+
+Check the box "Add Anaconda to my Path environment variable"
+
+### step 2 : Create your environment 
+
+#### Open Anaconda Prompt
```
Comment (josephsalmon) [inline Windows/Debbug path conda windows.md]: #### : could be ### only?

File: Windows/Debbug path conda windows.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Write the following commands :
+
+    conda create -n hmma238 matplotlib
+
+    conda activate hmma238
+
+### Step 3 : Check if it works
+
+Try to lunch in VsCode the file test_python.py (located in the git repository HMMA238 (Courses-IDE))
```
Comment (josephsalmon) [inline Windows/Debbug path conda windows.md]: typos:
lunch -> launch

correct path:

`../HMMA238/Courses/IDE`

**Review by josephsalmon — COMMENTED** [review]
I have make a thorough feedback to improve your contribution.
Mostly it is cosmetic/typos.
If you can address the comment, I am fine with merging.
Thx again!

**Comment by Poncheele** [comment]
I think it's all good now.

**Comment by josephsalmon** [comment]
It looks like it. Thanks for the effort!

**Pull request merged**